### ISSUE TITLE
Incompatible with the current GoLand version

### ISSUE THREAD

**Issue by gbedoya** [golangci/golangci-lint#965]
The GoLand 'Event Log' is reporting that plugin 'Go Linter' is incompatible with the latest version **2019.3.1**

Details:
```
Plugin Error: Plugin "Go Linter" is incompatible (target build range is 183.* to 192.*).
```

**Comment by ernado** [comment]
Sorry, seems like we can't fix it because actually it is 3rd-party plugin.
I've tried to find a github link of https://plugins.jetbrains.com/plugin/12496-go-linter but no results.

**Comment by gbedoya** [comment]
The link you provided mentions ...  "The plugin update is pending JetBrains approval"

This wasn't there earlier today.
